.golangci.yml

```diff
@@ -56,6 +56,9 @@ issues:
     - path: scale_handler.go
       linters:
         - gocyclo
+    - path: _finalizer.go
```

Comment:
Could you please add a comment with a description why are we adding this exception, eg. with a link to this PR?

Reply:
Sure thing! I missed that you had already given an example